CHANGES.rst

```diff
@@ -7,6 +7,10 @@ New Features
 API Changes
 ^^^^^^^^^^^
 
+- All operations now accept Spectrum1D and Quantity-type images. All accepted
+image types are now processed internally as Spectrum1D objects. [#146]
+-

```

Comment:
i assume this will be filled in a bit more before merging.

Reply:
I can include that operations' `image` attributes will now always be `Spectrum1D` objects. Was there anything else you were thinking should be here? I thought many of the changes were too behind the scenes to make the changelog.